official/vision/modeling/backbones/resnet.py

```diff
@@ -429,4 +443,5 @@ def build_resnet(
       norm_momentum=norm_activation_config.norm_momentum,
       norm_epsilon=norm_activation_config.norm_epsilon,
       kernel_regularizer=l2_regularizer,
-      bn_trainable=backbone_cfg.bn_trainable)
+      bn_trainable=backbone_cfg.bn_trainable,
+      use_first_projection=backbone_cfg.use_first_projection)
```

Comment:
This requires backbone config having a new field `use_first_projection` to make it work. Please add it and be sure to make the new field optional for backward compatibility.

Reply:
I think this is done right [here](https://github.com/tensorflow/models/pull/10584/files#diff-7d1b79cdcc36ae1c1c5b50cab306205681ededfad0356ec0506e2280dd878cceR405), no? otherwise I am not sure what you mean